site/en/_data/stories.yml

```diff
@@ -2,8 +2,16 @@ featured:
   modifiers:
     custom: landing--stories-full-bleed
 sections:
+  headings:
+    - tag: pwa
+      title: Progressive Web Applications
+      subtitle:
+    - tag: games
+      title: Games
+      subtitle:
+    - tag: android
+      title: Android
+      subtitle:
```

Comment:
This is great! Can you also do the same for Spanish (`es`)? The big difference is that the tag for `games` (and the title) should be `juegos`

Reply:
Done.